db/controllers/endpoints.controllers.js

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs/promises");
+
+exports.getEndpoints = (req, res, next) => {
+  return fs
+    .readFile(
```

Comment:
as we are reading a file here, we should have a model to handle this. Another option is you could directly require in the endpoints.json data and send this to the client without having to read the file :)